docs/terminology.md

```diff
@@ -0,0 +1,178 @@
+---
+id: terminology
+title: Temporal terminology
+sidebar_label: Terminology
+---
+
+Temporal conceptualizes software development in a unique way. Few products share overlapping methodologies and concepts with Temporal.
+
+Therefore, the following terms are re-defined within the context of the Temporal product, and used throughout the documentation and reference material to describe aspects of it.
+
+### Activity
+
+A business-level function that implements your application logic, such as calling a service or transcoding a media file.
```

Comment:
@flossypurse: One other change that isn't obvious because of the wholesale movement of text from Glossary to Terminology: I regularized the first sentence of each definition to be a definition-style incomplete sentence. Previously, we had a mix of complete and incomplete sentences.

Reply:
Nice!
Good call.